### PR TITLE
[node-manager][doc] review the description of minPerZone and maxPerZone

### PR DESCRIPTION
**Pull request by kirkonru**
## Description
Review the description of `minPerZone` and `maxPerZone` relations in the NodeGroup CR documentation.

## Why do we need it, and what problem does it solve?
Update the documentation to make it more user-friendly.

## What is the expected result?
The documentation becomes easier to understand.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: fix
summary: "Update the description of `minPerZone` and `maxPerZone` in the NodeGroup CR documentation."
impact_level: low
```

### CODE REVIEW

File: candi/openapi/doc-ru-node_group.yaml [hunk 1]
```diff
@@ -147,12 +147,12 @@ spec:
                       description: |
                         Минимальное количество инстансов в зоне. Проставляется в объект `MachineDeployment` и в качестве нижней границы в cluster autoscaler.
 
-                        > **Внимание!** `minPerZone` != 0 и `minPerZone` <= `maxPerZone` или `minPerZone` == `maxPerZone` == 0.
+                        > **Внимание!** Минимальное количество инстансов может варьироваться от единицы до `maxPerZone` (`minPerZone` != 0 и `minPerZone` <= `maxPerZone`), либо может быть равно нулю — в этом случае максимальное количество инстансов в зоне также должно быть равно нулю (`minPerZone` == `maxPerZone` == 0).
```
Comment (kirkonru) [inline candi/openapi/doc-ru-node_group.yaml:150]:
```suggestion
                        > **Внимание!** Минимальное количество инстансов (`minPerZone`) может принимать значение от единицы до `maxPerZone` (0 < `minPerZone` <= `maxPerZone`), либо `minPerZone` и `maxPerZone` могут быть оба равны нулю (`minPerZone` = `maxPerZone` = 0).
```

File: candi/openapi/node_group.yaml [hunk 2]
```diff
@@ -233,7 +233,7 @@ spec:
 
                         This value is used in the MachineDeployment object and as a lower bound in cluster-autoscaler.
 
-                        > **Caution!** `minPerZone` != 0 and `minPerZone` <= `maxPerZone` or `minPerZone` == `maxPerZone` == 0.
+                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (`minPerZone` !=0 and `minPerZone` <= `maxPerZone`), or it may be equal to zero, in which case the maximum number of instances in the zone must also be equal to zero (`minPerZone` == `maxPerZone` == 0).
```
Comment (kirkonru) [inline candi/openapi/node_group.yaml:236]:
```suggestion
                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (0 < `minPerZone` <= `maxPerZone`), or `minPerZone` and `maxPerZone` may both be equal to zero (`minPerZone` = `maxPerZone` = 0).
```

File: candi/openapi/node_group.yaml [hunk 3]
```diff
@@ -242,7 +242,7 @@ spec:
 
                         This value is used as the upper bound in cluster-autoscaler.
 
-                        > **Caution!** `minPerZone` != 0 and `minPerZone` <= `maxPerZone` or `minPerZone` == `maxPerZone` == 0.
+                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (`minPerZone` !=0 and `minPerZone` <= `maxPerZone`), or it may be equal to zero, in which case the maximum number of instances in the zone must also be equal to zero (`minPerZone` == `maxPerZone` == 0).
```
Comment (kirkonru) [inline candi/openapi/node_group.yaml:245]:
```suggestion
                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (0 < `minPerZone` <= `maxPerZone`), or `minPerZone` and `maxPerZone` may both be equal to zero (`minPerZone` = `maxPerZone` = 0).
```

File: candi/openapi/node_group.yaml [hunk 4]
```diff
@@ -857,7 +857,7 @@ spec:
 
                         This value is used in the MachineDeployment object and as a lower bound in cluster-autoscaler.
 
-                        > **Caution!** `minPerZone` != 0 and `minPerZone` <= `maxPerZone` or `minPerZone` == `maxPerZone` == 0.
+                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (`minPerZone` !=0 and `minPerZone` <= `maxPerZone`), or it may be equal to zero, in which case the maximum number of instances in the zone must also be equal to zero (`minPerZone` == `maxPerZone` == 0).
```
Comment (kirkonru) [inline candi/openapi/node_group.yaml:860]:
```suggestion
                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (0 < `minPerZone` <= `maxPerZone`), or `minPerZone` and `maxPerZone` may both be equal to zero (`minPerZone` = `maxPerZone` = 0).
```

File: candi/openapi/node_group.yaml [hunk 5]
```diff
@@ -866,7 +866,7 @@ spec:
 
                         This value is used as the upper bound in cluster-autoscaler.
 
-                        > **Caution!** `minPerZone` != 0 and `minPerZone` <= `maxPerZone` or `minPerZone` == `maxPerZone` == 0.
+                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (`minPerZone` !=0 and `minPerZone` <= `maxPerZone`), or it may be equal to zero, in which case the maximum number of instances in the zone must also be equal to zero (`minPerZone` == `maxPerZone` == 0).
```
Comment (kirkonru) [inline candi/openapi/node_group.yaml:869]:
```suggestion
                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (0 < `minPerZone` <= `maxPerZone`), or `minPerZone` and `maxPerZone` may both be equal to zero (`minPerZone` = `maxPerZone` = 0).
```

File: candi/openapi/node_group.yaml [hunk 6]
```diff
@@ -1456,7 +1456,7 @@ spec:
 
                         This value is used in the MachineDeployment object and as a lower bound in cluster-autoscaler.
 
-                        > **Caution!** `minPerZone` != 0 and `minPerZone` <= `maxPerZone` or `minPerZone` == `maxPerZone` == 0.
+                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (`minPerZone` !=0 and `minPerZone` <= `maxPerZone`), or it may be equal to zero, in which case the maximum number of instances in the zone must also be equal to zero (`minPerZone` == `maxPerZone` == 0).
```
Comment (kirkonru) [inline candi/openapi/node_group.yaml:1459]:
```suggestion
                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (0 < `minPerZone` <= `maxPerZone`), or `minPerZone` and `maxPerZone` may both be equal to zero (`minPerZone` = `maxPerZone` = 0).
```

File: candi/openapi/node_group.yaml [hunk 7]
```diff
@@ -1465,7 +1465,7 @@ spec:
 
                         This value is used as the upper bound in cluster-autoscaler.
 
-                        > **Caution!** `minPerZone` != 0 and `minPerZone` <= `maxPerZone` or `minPerZone` == `maxPerZone` == 0.
+                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (`minPerZone` !=0 and `minPerZone` <= `maxPerZone`), or it may be equal to zero, in which case the maximum number of instances in the zone must also be equal to zero (`minPerZone` == `maxPerZone` == 0).
```
Comment (kirkonru) [inline candi/openapi/node_group.yaml:1468]:
```suggestion
                        > **Caution!** The minimum number of instances may range from one to `maxPerZone` (0 < `minPerZone` <= `maxPerZone`), or `minPerZone` and `maxPerZone` may both be equal to zero (`minPerZone` = `maxPerZone` = 0).
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [2932](https://github.com/deckhouse/deckhouse/pull/2932) to the branch [release-1.39](https://github.com/deckhouse/deckhouse/tree/release-1.39) successful!

**Pull request merged**